CONTRIBUTING.md

```diff
@@ -118,7 +118,7 @@ These documents provide guidance creating a well-crafted commit message:
 
 You can submit an issue or a bug to our [GitHub repository](https://github.com/vmware/clarity/issues).  You must provide:
 
-* The link to the reproduction scenario you created using [http://plnkr.co](http://plnkr.co)
+* The link to the reproduction scenario you created using the [Clarity Plunker Template](https://plnkr.co/edit/VPxyl7?p=preview)
```

Comment:
Should we say "If possible please provide the link ..." just like you did in README.md's feedback section?